app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  # chuyển email về dạng chữ in thường
```

Comment:
Remove ALL comments in this file. Make a note or something that out of your project to keep your knowledges

Reply:
Dạ vâng anh